api-playground/troubleshooting.mdx

```diff
@@ -69,7 +69,7 @@ If your API pages aren't displaying correctly, check these common configuration
     requests, then all of these requests will fail. To fix this, configure your reverse proxy to
     allow `POST` requests to the `/api/request` path.
 
-    Alternatively, if your reverse proxy prevents you from accepting `POST` requests, you can configure Mintlify to send requests directly to your backend with the `api.playground.proxy` setting in the `docs.json`, as described [here](settings#api-configurations). This will
+    Alternatively, if your reverse proxy prevents you from accepting `POST` requests, you can configure Mintlify to send requests directly to your backend with the `api.playground.proxy` setting in the `docs.json`, as described [here](https://mintlify.com/docs/settings#param-proxy). This will
```

Comment:
```suggestion
    Alternatively, if your reverse proxy prevents you from accepting `POST` requests, you can configure Mintlify to send requests directly to your backend with the `api.playground.proxy` setting in the `docs.json`. See [Settings](https://mintlify.com/docs/settings#param-proxy) for more information. This will
```

An option to make the link text more descriptive